edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/ActiveReeferSettings.java

```diff
@@ -59,43 +55,26 @@ public class ActiveReeferSettings {
   @Column(name = "is_bulb_mode", nullable = false)
   private Boolean isBulbMode = Boolean.FALSE;
 
-  @Builder.Default
-  @Column(name = "is_gen_set_required", nullable = false)
-  private Boolean isGeneratorSetRequired = Boolean.FALSE;
-
-  @Builder.Default
-  @Column(name = "is_pre_cooling_required", nullable = false)
-  private Boolean isPreCoolingRequired = Boolean.FALSE;
-
-  @Builder.Default
-  @Column(name = "is_cold_treatment_required", nullable = false)
-  private Boolean isColdTreatmentRequired = Boolean.FALSE;
+  @Column(name = "temperature_setpoint", nullable = false)
+  private Float temperatureSetpoint;
 
-  @Builder.Default
-  @Column(name = "is_hot_stuffing_allowed", nullable = false)
-  private Boolean isHotStuffingAllowed = Boolean.FALSE;
+  @Column(name = "temperature_unit", nullable = false)
+  @Enumerated(EnumType.STRING)
+  private TemperatureUnit temperatureUnit;
 
-  @Builder.Default
-  @Column(name = "is_tracing_required", nullable = false)
-  private Boolean isTracingRequired = Boolean.FALSE;
+  @Column(name = "o2_setpoint")
+  private Float o2Setpoint;
 
-  @Builder.Default
-  @Column(name = "is_monitoring_required", nullable = false)
-  private Boolean isMonitoringRequired = Boolean.FALSE;
+  @Column(name = "co2_setpoint")
```

Comment:
I added min=0 and max=100 to this in the API

Reply:
Added those validations in the TO: https://github.com/dcsaorg/DCSA-Edocumentation/pull/83/commits/a6948c2d2d4ed0c6cd1e07bae049c955e787b6d2